lib/phoenix_live_reload/channel.ex

```diff
@@ -30,9 +31,11 @@ defmodule Phoenix.LiveReloader.Channel do
       ext = Path.extname(path)
 
       for {path, ext} <- [{path, ext} | debounce(debounce, [ext], patterns)] do
-        asset_type = remove_leading_dot(ext)
-        Logger.debug("Live reload: #{Path.relative_to_cwd(path)}")
-        push(socket, "assets_change", %{asset_type: asset_type})
+        if ChangeTracker.put_new_hash(path) do
```

Comment:
Debounce is ignoring a bunch of files already that match the same extension, so this would have to be moved inside the debounce code. :)